msal/src/test/java/com/microsoft/identity/client/robolectric/utils/AcquireTokenTestHelper.java

```diff
@@ -67,6 +69,28 @@ public void onCancel() {
         return callback;
     }
 
+    public static AuthenticationCallback failedSilentRequestDuplicateCommand() {
```

Comment:
nit: Rename to `failedSilentRequestDuplicateCommandCallback`

Reply:
fixed.